Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3-buster
```

Comment:
why use buster? bullseye is the current Debian distribution.
why use 3 and not something more up to date like [3.10.11-bullseye](https://github.com/docker-library/python/blob/4c86fbef4c546a65e372c9bc8c27b8c17bfe42fd/3.10/bullseye/Dockerfile)

Reply:
Good point! Fixed now